java/pom.xml

```diff
@@ -74,7 +74,7 @@
         <maven-dependency-plugin.version>3.2.0</maven-dependency-plugin.version>
         <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
         <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
-        <powsybl-dependencies.version>1.3.1</powsybl-dependencies.version>
+        <powsybl-dependencies.version>1.4.0-SNAPSHOT</powsybl-dependencies.version>
```

Comment:
could you use integration branch please ?

Reply:
maybe it needs to be updated

Reply:
rebased to integration branch :)

Reply:
I removed this integration for CI tests